modin/utils.py

```diff
@@ -264,18 +265,22 @@ def _replace_doc(
     if (
         source_doc.strip()
         and apilink
-        and "`pandas API documentation for " not in target_doc
+        and "pandas API documentation for " not in target_doc
         and (not (attr_name or "").startswith("_"))
     ):
-        if attr_name:
-            token = f"{apilink}.{attr_name}"
-        else:
-            token = apilink
-        url = _make_api_url(token)
+        links = []
+        for link in apilink:
+            if attr_name:
+                token = f"{link}.{attr_name}"
+            else:
+                token = link
+            url = _make_api_url(token)
+            links.append(f"`{token} <{url}>`_")
```

Comment:
```suggestion
        links = [None] * len(apilink)
        for i, link in enumerate(apilink):
            if attr_name:
                token = f"{link}.{attr_name}"
            else:
                token = link
            url = _make_api_url(token)
            links[i] = f"`{token} <{url}>`_"
```

Reply:
Not sure i understood this change... Why do we need such "dummy" links?

Reply:
There is no dummy links here. This change is just to speed up things a bit. Since we know the size of `apilink` we can precompute the list of `links`.